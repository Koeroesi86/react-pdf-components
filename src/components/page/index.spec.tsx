import React from "react";
import { renderToString, Document } from "@react-pdf/renderer";
import Page from "./index";

describe("<Page/>", () => {
  it("should render", async () => {
    const result = await renderToString(
      <Document>
        <Page/>
      </Document>
    );
    expect(result).toMatchSnapshot();
  });
});
