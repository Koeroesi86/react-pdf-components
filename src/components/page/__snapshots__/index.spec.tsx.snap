// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page/> should render 1`] = `
"%PDF-1.3
%����
8 0 obj
<<
/Type /ExtGState
/ca 1
>>
endobj
7 0 obj
<<
/Type /Page
/Parent 1 0 R
/MediaBox [0 0 595.280029 841.890015]
/Contents 5 0 R
/Resources 6 0 R
>>
endobj
6 0 obj
<<
/ProcSet [/PDF /Text /ImageB /ImageC /ImageI]
/ExtGState <<
/Gs1 8 0 R
>>
>>
endobj
5 0 obj
<<
/Length 351
>>
stream
1 0 0 -1 0 841.890015 cm
q
q
0 0 m
595.280029 0 l
595.280029 0 595.280029 0 595.280029 0 c
595.280029 841.890015 l
595.280029 841.890015 595.280029 841.890015 595.280029 841.890015 c
0 841.890015 l
0 841.890015 0 841.890015 0 841.890015 c
0 0 l
0 0 0 0 0 0 c
h
W n
/Gs1 gs
/DeviceRGB cs
1 1 1 scn
0 0 595.280029 841.890015 re
f
Q
q
1 0 0 1 0 0 cm
Q
Q

endstream
endobj
10 0 obj
(react-pdf)
endobj
11 0 obj
(react-pdf)
endobj
12 0 obj
(D:20210414183756Z)
endobj
9 0 obj
<<
/Producer 10 0 R
/Creator 11 0 R
/CreationDate 12 0 R
>>
endobj
3 0 obj
<<
/Type /Catalog
/Pages 1 0 R
/Names 2 0 R
/ViewerPreferences 4 0 R
>>
endobj
1 0 obj
<<
/Type /Pages
/Count 1
/Kids [7 0 R]
>>
endobj
2 0 obj
<<
/Dests <<
  /Names [
]
>>
>>
endobj
4 0 obj
<<
/DisplayDocTitle true
>>
endobj
xref
0 13
0000000000 65535 f 
0000000928 00000 n 
0000000985 00000 n 
0000000841 00000 n 
0000001032 00000 n 
0000000272 00000 n 
0000000177 00000 n 
0000000059 00000 n 
0000000015 00000 n 
0000000766 00000 n 
0000000674 00000 n 
0000000702 00000 n 
0000000730 00000 n 
trailer
<<
/Size 13
/Root 3 0 R
/Info 9 0 R
>>
startxref
1075
%%EOF
"
`;
